

diff --git a/C:/Users/pekka/koodit/patchmanager/smooth-remorse-counters/sfos_4.0.1.45/orig/RemorseBase.qml b/C:/Users/pekka/koodit/patchmanager/smooth-remorse-counters/sfos_4.0.1.45/modified/RemorseBase_mod_4.0.1.45.qml
index .. 100644
--- a/C:/Users/pekka/koodit/patchmanager/smooth-remorse-counters/sfos_4.0.1.45/orig/RemorseBase.qml
+++ b/C:/Users/pekka/koodit/patchmanager/smooth-remorse-counters/sfos_4.0.1.45/modified/RemorseBase_mod_4.0.1.45.qml
@@ -101,7 +101,7 @@
             }
 
             radius: background.radius
-            width: Math.round((_seconds - _secsRemaining)/_seconds * background.width)
+            width: (_timeout - _msRemaining)/_timeout * background.width // patch smooth-remorse-counters
             roundedCorners: _secsRemaining === 0 ? Corners.All
                                                  : Corners.TopRight | Corners.BottomRight
 
